src/prebid.js

```diff
@@ -342,11 +342,11 @@ $$PREBID_GLOBAL$$.clearAuction = function() {
 };
 
 /**
- *
- * @param bidsBackHandler
- * @param timeout
- * @param adUnits
- * @param adUnitCodes
+ * @property {Object} defaults
+ * @property {function} defaults.bidsBackHandler
+ * @property {number} defaults.timeout
+ * @property {Array} defaults.adUnits
+ * @property {Array} defaults.adUnitCodes
```

Comment:
I'm not positive, but... is this generating the right docs? I wouldn't expect to see `@property` used in this way.

A sample call would look something like:

```
pbjs.requestBids({
  timeout: ...,
  adUnits: ...,
  adUnitCodes: ...,
  bidsBackHandler: function(bids) { ... }
});
```

If so, then cool ^^.

Reply:
I was trying to figure that out - mostly I wanted to shut up the linter.  I don't think (?) it's generating the right docs - it's generating the following:

```
### requestBids

**Parameters**

-   `$0` **[Object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)**  (optional, default `{}`)
    -   `$0.bidsBackHandler`  
    -   `$0.timeout`  
    -   `$0.adUnits`  
    -   `$0.adUnitCodes`  
-   `bidsBackHandler`  
-   `timeout`  
-   `adUnits`  
-   `adUnitCodes`  
```
I don't know if that is the right doc structure b/c I don't understand what this way of referring to arguments means:

`requestBids = function ({ bidsBackHandler, timeout, adUnits, adUnitCodes } = {})`

Does it mean that: 
1. This defaults to an empty object?
2. That providing the object with the right keys as input is optional?  Because it doesn't seem optional. (?)
3. Something else?

Reply:
I tested a bit... I think what you want is to swap the `property` tags here with `param`. That makes more reasonable docs (screenshotted below)

![screen shot 2017-08-30 at 9 21 14 am](https://user-images.githubusercontent.com/27972385/29874393-9f6ebb24-8d64-11e7-9b40-adf5bc086a69.png)

See also: http://usejsdoc.org/tags-param.html#parameters-with-properties

Reply:
Ah, makes sense after reading the doc.  OK, just switched back to `param`.

After changing back to `param`,  `documentation lint` still throwing one warning, but the message reads like its parser just isn't smart enough to understand the code/JSDoc, not like there is an actual problem, so it seems like we can safely ignore it:

```
warning  An explicit parameter named defaults was specified but didn't match inferred information $0
```

Reply:
oh that's super annoying...

That sounds like an error meant to prevent mismatched names, like this:

```
/**
 * @param e ...
 */
function handleError(error) { ... }
```

My guess is that `$0` is just the placeholder value they assign because the fancy ES6 syntax doesn't actually give the param a name.

see: https://github.com/documentationjs/documentation/issues/270